DaemonCommander/DaemonCommander.cs

```diff
@@ -161,7 +161,7 @@ private string SendCommand(string command)
 		{
 			try
 			{
-				var resp = new byte[2048];
+				var resp = new byte[16 * 1048];

```

Comment:
We should really change the protocol. Either to size + payload, http, grpc or something else.

Reply:
Even with size + payload, we'll need to only accept messages up to a given size, so similar bugs could happen (maximum size too small needing to be raised).

Reply:
we can accept an int, then accept sizeof(the previous int). This what I do in gvisor-tap-vsock.

Reply:
But we cannot blindly trust this int, we don't want to blindly do 2GB allocations because some other process told us to do so.

Reply:
Btw, `1048` should be `1024` or `2048` I guess?

Reply:
> which means the same issue is still present.

but for now less likely to encounter, in need of a follow-up permanent fix.
suggestions?

Reply:
I thought it'd be better to move to the HTTP protocol, once its merged in crc () for now thought we need to increase the size to temporarily fix the bug.

Reply:
Yup, seems quick enough to have this fix for now, with the http work as a better fix later.